demo/src/app/pages/modules/datepicker/datepicker.page.ts

```diff
@@ -112,6 +119,13 @@ export class DatepickerPage {
                     defaultValue: "0"
                 },
                 {
+                    name: "pickerInitialView",
+                    type: "CalendarViewType",
+                    description: "Specifies the initial view for the datepicker. Options are: <code>CalendarViewType.Year</code>, " +
```

Comment:
alas TS 2.4.1 support still isn't in Angular when using strict null checks so the "enum" solution is still the best way